site-src/geps/gep-1709.md

```diff
@@ -37,8 +45,295 @@ which implementations can prove they satisfy and be recognized for.
 
 ## API
 
-TODO
+The API for conformance profiles will be a pipeline that implementations can
+opt into. The workflow is effectively:
+
+1. select a [profile](#profiles)
+2. [integrate](#integration) tests in the downstream project
+3. [report results and get certified](#certification)
+
+The goal is to make selecting a conformance profile as simple of a process as
+feasible and support both the existing command line integration approach as well
```

Comment:
what is the existing command line approach? 
`go test  ......`?

Reply:
Yes that's correct, we can add that here:

```suggestion
feasible and support both the existing command line integration approach (e.g. `go test`) as well
```